.github/workflows/nightly_test.yml

```diff
@@ -147,3 +151,34 @@ jobs:
           path: src/testlog/*
           if-no-files-found: error
           retention-days: 14
+
+      - name: SHA info
+        id: sha-info
+        if: failure()
+        run: |
+          echo ${GITHUB_REF#refs/heads/}-${GITHUB_SHA}
+          echo SHA=${GITHUB_REF#refs/heads/}-${GITHUB_SHA} >> $GITHUB_OUTPUT
+          echo RUN_URL=${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}  >> $GITHUB_OUTPUT
+          echo COMMIT_URL=${{ github.server_url }}/${{ github.repository }}/commit/${GITHUB_SHA} >> $GITHUB_OUTPUT
+
+      - name: Send Github Action failure to Slack
```

Comment:
should we just make this an action that takes a title and some URLs/SHAs/whatever? that way it's more of a template we can use?